docs/Using-SRIOV-and-Clear-Containers.md

```diff
@@ -0,0 +1,218 @@
+# Setup to use SRIO-V with Clear Containers
+
+Single Root IO Virtualization enables splitting a physical device into
+virtual functions (VF) which enables direct pass through to virtual machines or
+containers.  For Clear Containers, we have enabled a CNM plugin and necessary
+changes to facilitate making use of SRIO-V for network based devices.
+
+## Install the SRIO-V Docker\* plugin
+
+To create a network with VFs associated with it which can be passed to
+Clear Containers, install a SRIOV Docker plugin. The created network will be
+based on a physical function (PF) device, and can create 'n' containers, where
+'n' is the number of VFs associated with the PF
+
+To install the plugin, you can follow the [plugin installation instructions](https://github.com/clearcontainers/sriov).
+
+
+## Host setup for SRIO-V
+
+You may need to rebuild your system's kernel in order to disable VFNOIOMMU in
```

Comment:
- s/your system's kernel/your host system's kernel/
- The kernel config option appears to be `CONFIG_VFIO_NOIOMMU`, not `VFNOIOMMU`.